webpack/components/AnsibleRoles/AnsibleRolesPage.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import TableIndexPage from 'foremanReact/components/PF4/TableIndexPage/TableIndexPage';
+import RelativeDateTime from 'foremanReact/components/common/dates/RelativeDateTime';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { getDocsURL } from 'foremanReact/common/helpers';
+import { getActions } from './AnsibleRolesPageHelpers';
+import { AnsibleRolesImportButtonWrapper } from './components/AnsibleRolesImportButtonWrapper';
+
+export const AnsibleRolesPage = () => (
+  <TableIndexPage
+    header={__('Ansible Roles')}
+    controller="ansible_roles"
+    apiUrl="/ansible/ui_ansible_roles"
+    apiOptions={{ key: 'ANSIBLE_ROLES_API_REQUEST_KEY' }}
+    hasHelpPage
+    creatable={false}
+    isDeleteable
+    searchable
```

Comment:
By default, the `searchable` property is already set to true. Do you think we should keep it here for readability?

Reply:
Removed